src/libawkward/forth/ForthMachine.cpp

```diff
@@ -3310,6 +3345,13 @@ namespace awkward {
               break;
             }
 
+            case CODE_STRING_ERROR: {
+              I string_num = bytecode_get();
+              bytecodes_pointer_where()++;
+              stack_push(string_num);
```

Comment:
Protect this by preceding `stack_push` with

```c++
              if (stack_cannot_push()) {
                current_error_ = util::ForthError::stack_overflow;
                return;
              }
```

Reply:
done